plugin-flex-ts-template-v2/src/feature-library/dispositions/custom-components/DispositionTab/DispositionTab.tsx

```diff
@@ -114,6 +138,49 @@ const DispositionTab = (props: OwnProps) => {
             ))}
           </RadioGroup>
         )}
+        {selectAttributes.map((attr) => {
+          const id = attr.conversation_attribute;
+          return (
+            <>
+              <Label htmlFor={id} required>
```

Comment:
These don't actually behave as `required`. Either the required flag should be removed or the logic in `CompleteTask` should be updated to validate the custom fields (I suppose it might also make sense to add settings per attribute as to whether each is required).

Reply:
Wrap Up form attributes can now optionally be marked as `required: true` via the config
CompleteTask will pop a notification when required fields are missing:
_"Please select an option or enter a value for all required fields"_